contracts/p1/StRSRVotes.sol

```diff
@@ -73,36 +86,39 @@ contract StRSRP1Votes is StRSRP1, IStRSRVotes {
         return pos == 0 ? 0 : _checkpoints[era][account][pos - 1].val;
     }
 
-    function getPastVotes(address account, uint256 blockNumber) public view returns (uint256) {
-        require(blockNumber < NetworkConfigLib.blockNumber(), "ERC20Votes: block not yet mined");
-        uint256 pastEra = _checkpointsLookup(_eras, blockNumber);
-        return _checkpointsLookup(_checkpoints[pastEra][account], blockNumber);
+    function getPastVotes(address account, uint256 timepoint) public view returns (uint256) {
+        require(timepoint < block.timestamp, "ERC20Votes: future lookup");
```

Comment:
We should add a test for this validation (to trigger `ERC20Votes: future lookup` Did not see it in the Governance test file (or StRSR test file not sure where these are checked)

Reply:
Yeah it used to be "block not mined yet" but changed it to this to be more accurate.